bgpd/bgp_errors.c

```diff
@@ -449,6 +449,18 @@ static struct log_ref ferr_bgp_err[] = {
 		.description = "The BGP flowspec subsystem has detected that there was a failure for installation/removal/modification of Flowspec from the dataplane",
 		.suggestion = "Gather log files from the router and open an issue, Restart FRR"
 	},
+	{
+		.code = EC_BGP_LINKSTATE_PACKET,
+		.title = "BGP Link-State packet processing error",
+		.description = "The BGP Link-State subsystem has detected a error in the send or receive of a packet",
+		.suggestion = "Gather log files from both sides of the peering relationship and open an issue"
+	},
+	{
+		.code = EC_BGP_LINKSTATE_INSTALLATION,
+		.title = "BGP Link-State Installation/removal Error",
+		.description = "The BGP Link-State subsystem has detected that there was a failure for installation/removal/modification of Link-State from the dataplane",
+		.suggestion = "Gather log files from the router and open an issue, Restart FRR"
+	},
```

Comment:
This error is not used. remove it please.

Reply:
removed